lib/games_collection.rb

```diff
@@ -12,4 +12,16 @@ def generate_objects_from_csv(csv_games_path)
     end
     objects
   end
+
+  def highest_total_score
+    @games.map do |game|
+      game.away_goals.to_i + game.home_goals.to_i
+    end.max
+  end
+
+  def lowest_total_score
+    @games.map do |game|
+      game.away_goals.to_i + game.home_goals.to_i
+    end.min
+  end
```

Comment:
I like these methods. I think we should get them down to one line to save space. After we merge to Daniels point we can refactor and include helper methods. then we can see what helper methods look similar and we can then make them into module's